pt_dummy_data.sql

```diff
@@ -14,9 +14,9 @@ INSERT INTO fachrichtung (bezeichnung)
 VALUES
     ('WI'), ('Etechnik'), ('BWL');
 
-INSERT INTO modul (fk_fachrichtung, modulnummer, beschreibung)
+INSERT INTO modul (id, fk_fachrichtung, modulnummer, beschreibung)
```

Comment:
wieso Spalte "id" wenn die nicht in den nachfolgenden Zeilen 19-22 nicht gefüllt wird?

Reply:
ah vergessen rauszunehmen